ta/pkcs11/src/processing_asymm.c

```diff
@@ -903,3 +933,159 @@ enum pkcs11_rc do_asymm_derivation(struct pkcs11_session *session,
 
 	return rc;
 }
+
+static enum pkcs11_rc wrap_rsa_aes_key(struct active_processing *proc,
+				       void *data, uint32_t data_sz,
+				       void *out_buf, uint32_t *out_sz)
+{
+	TEE_Result res = TEE_ERROR_GENERIC;
+	TEE_Attribute tee_attrs[2] = { };
+	struct rsa_aes_key_wrap_processing_ctx *ctx = proc->extra_ctx;
+	mbedtls_nist_kw_context kw_ctx;
+	uint8_t aes_key_value[32] = { };
+	uint32_t aes_key_size = ctx->aes_key_bits / 8;
+	uint32_t tee_attrs_count = 0;
+	uint32_t aes_wraped_size = *out_sz;
```

Comment:
typo?

```suggestion
	uint32_t aes_wrapped_size = *out_sz;
```

Reply:
right. Ditto for `wraped_key_size` in `unwrap_rsa_aes_key()`.

Reply:
ok